ResponseDetective.xcodeproj/project.pbxproj

```diff
@@ -873,55 +873,63 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 3A7D01861CD3BE0100B7FBD7 /* Framework-macOS-Debug.xcconfig */;
 			buildSettings = {
+				SWIFT_VERSION = 3.0;
```

Comment:
Please remove these `SWIFT_VERSION` settings from the project file itself (by hitting backspace in "Build Settings" list) and update the value in [ResponseDetective-Common.xcconfig](https://github.com/netguru/ResponseDetective/blob/develop/Configuration/ResponseDetective-Common.xcconfig) instead.